src/components/cap-nav.js

```diff
@@ -170,102 +201,123 @@ class CapNav extends LitElement {
 		}
 	}
 
-	_openSearchInterstitial(event) {
-		event.preventDefault();
+	get searchInterstitial() {
+		return this.shadowRoot.querySelector(".searchInterstitial");
+	}
+
+	_openSearchInterstitial() {
+		this.searchInterstitial.showModal();
+		console.log(this.butts);
+	}
 
-		const searchInterstitial = this.shadowRoot.querySelector(
-			".searchInterstitial",
-		);
-		searchInterstitial.showModal();
+	_closeSearchInterstitial() {
+		this.searchInterstitial.close();
 	}
 
 	render() {
 		return html`
-			<div>
```

Comment:
This was probably a remnant from past development efforts—it's no longer necessary, so I removed it.